src/compo/viirs_biaswriter.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import netCDF4
+import numpy as np
+import os
+
+parser = argparse.ArgumentParser(
+    description=('Write VIIRS aerosol optical depth bias coefficients to NetCDF')
+)
+parser.add_argument(
+    '-o', '--output',
+    help="name of NetCDF output file",
+    type=str, required=True)
+
+args = parser.parse_args()
```

Comment:
Please add some brief comments to help the next developer. For example, it looks like you are starting with a basic setting of constant coefficients (pardon my ignorance of how the bias coefficients actually work in JEDI - it's not my area of expertise). Are there plans to expand this in the future to something more sophisticated, and if so how does that work (matrix instead of vector for example). Also, where did the numbers on lines 31, 32 come from? Information like this would help the next person that is tasked to expand/modify this writer.

I've just made a couple suggestions which might be a little off target for helping the next developer, and you likely have a better idea of what might be useful in this regard so please use your discretion.

Thanks!

Reply:
Sure. I can add something.